### ISSUE TITLE
Replace thumbnails for PSU Important Farmland maps

### ISSUE THREAD

**Issue by karenmajewicz** [geobtaa/geoportal#587]
I originally uploaded the items in this [collection](https://geo.btaa.org/?f%5Bpcdm_memberOf_sm%5D%5B%5D=2787a9e8-0bef-452e-9e5f-e83042c193b4&f%5Bschema_provider_s%5D%5B%5D=Pennsylvania+State+University) with thumbnail links that don't work.

I then removed the bad links and added IIIF image APIs instead. Can we clear the originally links and set these to reharvest thumbnails using the IIIF API instead?

**Comment by ewlarson** [comment]
GeoBlacklight Sidecar Images is closely tied to the `viewer_protocol` value for each SolrDocument, which here is `iiif_manifest`, but we don't have support for thumbnails from IIIF Manifests.

This item, and others like it, are being "placeheld", as the thumbnail process returns this transaction:

`{"solr_doc_id"=>"pst_0083368041", "solr_version"=>1796518337326350336, "placeheld"=>true, "viewer_protocol"=>"iiif_manifest", "error"=>"service_url NameError", "image_url"=>nil, "gblsi_thumbnail_uri"=>false}`

A couple options for these type of items...

1. Check all potential thumbnail generation candidates — include regular iiif here
2. Generate a thumnail from a IIIF Manifest file from the first canvas

**Comment by karenmajewicz** [comment]
All the items in this set do have an Image API in addition to the Presentation API. Could the image harvest code be configured to look for the Image API first?

**Comment by ewlarson** [comment]
Yeah, we need to enhance GBL Sidecar Images to better handle IIIF Manifests and to also check for other potential thumbnail options if the default viewer protocol doesn't pan out.

**Comment by karenmajewicz** [comment]
It appears that most of these have harvested, but they are not showing up in the search results.

**Comment by ewlarson** [comment]
Yeah, we are still presenting the old ActiveStorage thumbnails in production (wanted to make sure the initial bulk harvest was successful). We need to switch over to Kithe/Shrine.rb/Amazon thumbnails.
